frontend/test/jest-setup.js

```diff
@@ -16,7 +16,6 @@ process.on("uncaughtException", err =>
 
 if (process.env["DISABLE_LOGGING"] || process.env["DISABLE_LOGGING_FRONTEND"]) {
   global.console = {
-    ...console.log,
```

Comment:
The author likely wanted to spread `console` since `{...console.log}` results in an empty object. That means this line has been doing nothing.